src/core/game-camera.js

```diff
@@ -21,8 +24,8 @@ export class GameCamera {
     // We need to move the mouse by the same amount as the camera.
     // Move the mouse first since we need to compute the camera delta
     // and the delta is lost once the camera is already moved.
-    gameMouse.x += x - this.#x
-    gameMouse.y += y - this.#y
+    gameMouse.x += xRound - this.#x
+    gameMouse.y += yRound - this.#y
```

Comment:
This was part of the issue with the mouse debug readout. The mouse position was shifted by the exact camera offset while the camera was shifted by a rounded number of pixels (to avoid sub-pixel rendering issues.)